services/cd-service/Buildfile.yaml

```diff
@@ -12,3 +12,4 @@ spec:
   - ../../pkg
   - ../../go.mod
   - ../../go.sum
+  - ../../tests/integration-tests/
```

Comment:
This dependency looks the wrong way round, doesn't it?

Reply:
I wanted to express `if integration-tests changed, then build all services`.

I would like to refactor this dependency handling more in story SRX-8HW1TP. I would leave it as it is for now.